src/application/release-event-handler.ts

```diff
@@ -142,6 +132,35 @@ export class ReleaseEventHandler {
       }
     };
 
+  private async determineReleaser(
+    releaseAuthor: string,
+    rulesetRepo: RulesetRepository
+  ): Promise<User> {
+    // Use the GH release author unless a fixedReleaser is configured
+    let releaserUsername = releaseAuthor;
+    if (rulesetRepo.config.fixedReleaser) {
+      releaserUsername = rulesetRepo.config.fixedReleaser.login;
+      console.log(`Overriding releaser to ${releaserUsername}`);
+    }
+
+    // Fetch the user from GitHub. Note that their email won't be availble
```

Comment:
their email does appear in all their commits, but I guess we don't have a commit to work from in this context without cloning something...

Reply:
We could scan the commit history of the ruleset repository, however there wouldn't be a deterministic link as commits don't store the github username, just the person's name and email address.